testing/e2e/app/routes/ButtonGroup/route.tsx

```diff
@@ -1,23 +1,90 @@
-import { ButtonGroup, IconButton } from '@itwin/itwinui-react';
+import { Button, ButtonGroup, Flex, IconButton } from '@itwin/itwinui-react';
 import { SvgPlaceholder } from '@itwin/itwinui-icons-react';
 import { useSearchParams } from '@remix-run/react';
+import React from 'react';
```

Comment:
nit: always use `* as React` namespace import, not default import

same comment applies to all other instances

Reply:
we have had this rule for a while now, it's in the style guide

the default `import React` is considered legacy at this point, it's from before hooks and before the new JSX transform was introduced. i believe the React team wants to remove it in a future version, so they encourage namespace (`import * as React`) or named imports (`import {} from`)